python/composio/tools/toolset.py

```diff
@@ -977,6 +978,47 @@ def get_entity(self, id: t.Optional[str] = None) -> Entity:
         """Get entity object for given ID."""
         return self.client.get_entity(id=id or self.entity_id)
 
+    def get_expected_params(
+        self,
+        app: t.Optional[AppType] = None,
+        auth_scheme: t.Optional[
+            t.Literal[
+                "OAUTH2",
+                "OAUTH1",
+                "API_KEY",
+                "BASIC",
+            ]
+        ] = None,
+    ) -> t.List[AuthSchemeField]:
+        auth_schemes = {
+            scheme.auth_mode: scheme.fields
+            for scheme in self.client.apps.get(name=str(app)).auth_schemes or []
+        }
+
+        if auth_scheme is not None and auth_scheme not in auth_schemes:
+            raise ComposioSDKError(
+                message=f"Auth scheme `{auth_scheme}` not found for app `{app}`"
+            )
+
```

Comment:
Consider adding a type hint for the `app` parameter in the `initiate_connection` method. This would improve code readability and help catch potential type-related errors early.